the-collab-lab/tcl-36-smart-shopping-list
# Issue #23 - Refactoring and Creating a Utilities File

## Description

- Created a `utils.js` file to export general functions and other objects (e.g. `radioButtonOptions` and `navStyles`)
- Moved `setUpdateToDb` and `deleteItemFromDb` from component into `firebase.js`

## Related Issue

closes #23 

## Acceptance Criteria

-Make utility file and pull out functions and write comments for them as utilities

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

### After

Nothing to see here! As long as everything works... 🤞

## Testing Steps / QA Criteria

-Pull down `dj-util-refactoring` to your local environment and make sure everything works properly
-Feel free to add your suggestions as well!
